packages/amplify_core/lib/src/types/models/model.dart

```diff
@@ -1,75 +1,105 @@
-/*

```

Comment:
Same question.

We should try to avoid this if possible. This change will have to be pulled into next. I know this PR mostly touches files that are unchanged in next, but if we have to modify any of these files in next, it will be a nightmare to get this change in due to conflicts.

Reply:
Yea that's line ending change...
I separated everything in individual commits, please review by commits that would be more clear.

Reply:
I still see the same thing, though - were the line endings messed up before and you've fixed them or are they incorrect right now?

Reply:
Can you rebase on main? I've updated all the line endings so this PR should show just the diff after

Reply:
Done :) thanks for cleaning up the base.